deploy/prod.yaml

```diff
@@ -16,6 +16,9 @@ binderhub:
     hosts:
       - binder.pangeo.io
 
+  dind:
+    hostSocketDir: /var/run/dind/prod/docker.sock
```

Comment:
This should be /var/run/dind/prod, without the docker.sock, since it is only specifying the socket directory and not the socket itself.